board/freescale/mx6ullevk/mx6ullevk.c

```diff
@@ -276,12 +285,291 @@ int board_late_init(void)
 /*
  *
  */
-int checkboard(void)
-{
+int checkboard(void) {
 #ifdef CONFIG_SIKLU_BOARD
-	puts("Board: Siklu PCB19x\n");
+	unsigned int siklu_board_id = get_siklu_board_id();
+
+	switch (siklu_board_id) {
+		case SKL_BOARD_TYPE_PCB195:
+			puts("Board: Siklu PCB195\n");
+			break;
+		case SKL_BOARD_TYPE_PCB213:
+			puts("Board: Siklu PCB213\n");
+			break;
+		case SKL_BOARD_TYPE_PCB217:
+			puts("Board: Siklu PCB217\n");
+			break;
+		case SKL_BOARD_TYPE_PCB277:
+			puts("Board: Siklu PCB277\n");
+			break;
+		default:
+			puts("Board: Siklu Unknown\n");
+			break;
+	}
 #else
 	puts("Board: MX6ULL 14x14 EVK\n");
 #endif
 	return 0;
 }
+
+/* Called from ./common/board_f.c */
+int board_early_init_f(void)	/* NOT SPL */
+{
+	return 0;
```

Comment:
Just disable `CONFIG_BOARD_EARLY_INIT_F` to get rid of this empty stub.

Reply:
Done.